### ISSUE TITLE
Scheduler event wrapper element should have `aria-label`

### ISSUE THREAD

**Issue by matsbryntse** [bryntum/support#10500]
- [x] Set ariaLabel just in time, test that users can overwrite
- [x] Ensure users can set ariaLabel in eventRenderer in both Scheduler + Pro

Include resource, start date, duration (+ travel time when relevant)
